lib/adyen/client.rb

```diff
@@ -27,26 +27,39 @@ def env=(value)
       @env = value
     end
 
+    # remove 'https' from live_url_prefix if necessary
+    def live_url_prefix=(value)
+      if not value["https://"].nil?
+        value["https://"] = ""
+      end
+      @live_url_prefix = value
+    end
+
     # base URL for API given service and @env
     def service_url_base(service)
+      raise ArgumentError, "Please set Client.live_url_prefix to the portion of your merchant-specific URL prior to '-[service]-live.adyenpayments.com'" if @live_url_prefix.nil? and @env == :live
       if @env == :mock
         "http://localhost:#{@mock_port}"
       else
         case service
         when "Checkout"
-          if @env == :test
-            "https://checkout-test.adyen.com"
-          else
-            raise ArgumentError, "Please set Client.checkout_url_prefix to the portion of your merchant-specific URL prior to '-checkout-live'" if @checkout_url_prefix.nil?
-            "https://#{@checkout_url_prefix}-checkout-live.adyenpayments.com/checkout/services/PaymentSetupAndVerification"
-          end
+          url = "https://checkout-#{@env}.adyen.com/checkout/services/PaymentSetupAndVerification"
```

Comment:
@rikterbeek, I beleave there must be 

```ruby
"https://checkout-#{@env}.adyen.com/services/PaymentSetupAndVerification"
```
instead of
```ruby
"https://checkout-#{@env}.adyen.com/checkout/services/PaymentSetupAndVerification"
```